### ISSUE TITLE
QMenu: Screen reader unable to navigate contents [Accessibility]

### ISSUE THREAD

**Issue by TotallyMehis** [quasarframework/quasar#15279]
### What happened?

NVDA Screen reader is unable to navigate the contents of QMenu using arrow keys. Tested with latest Firefox & Chrome.

![explorer_QAjSkuYq6E](https://user-images.githubusercontent.com/4648654/212668732-a694a422-9438-48a8-ad4e-3ced243719fc.gif)

### What did you expect to happen?

For the screen reader to be able to navigate and read the options using arrow keys.

### Reproduction URL

https://codepen.io/TotallyMehis/pen/bGjrxjW

### How to reproduce?

1. Start NVDA screen reader.
2. Navigate to reproduction.
3. Press the menu button and attempt to navigate the list using arrow keys.

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

### Platforms/Browsers

Firefox, Chrome

**Comment by pdanpdan** [comment]
The default navigation is using tab key.
For the rest basically you control what is inside the menu.
If you need arrow key navigation inside the QList you can use a tab key navigation directive for Vue.
